README.md

```diff
@@ -104,27 +104,40 @@ The following needs to be done just once per setup.
     ```
 
 ### Linking
-1. In your application directory: Link the required OpenUI5 libraries
-    ```sh
-    npm link @openui5/sap.f
-    npm link @openui5/sap.m
-    npm link @openui5/sap.ui.core
-    npm link @openui5/themelib_sap_fiori_3
+
+Would you like to work on the application project and one or more of its UI5 framework dependencies at the same time? We got you covered!
+
+1. Create a new file `ui5-workspaces.yaml` in the root folder of the project, right next to the `ui5.yaml`
```

Comment:
ui5-workspace or ui5-workspaces ?

Reply:
It's `ui5-workspace.yaml`. Thanks a lot!